### ISSUE TITLE
bug of validating ruby, rp element

### ISSUE THREAD

**Issue by mtrootyy** [validator/validator#830]
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-rp-element
> Example
```
<ruby>
♥<rp>: </rp><rt>Heart</rt><rp>, </rp><rt lang=fr>Cœur</rt><rp>.</rp>
☘<rp>: </rp><rt>Shamrock</rt><rp>, </rp><rt lang=fr>Trèfle</rt><rp>.</rp>
✶<rp>: </rp><rt>Star</rt><rp>, </rp><rt lang=fr>Étoile</rt><rp>.</rp>
</ruby>
```

　　↓
　　
The result of validating the above;
>
```
Error: Text not allowed in element ruby in this context.
At line 10, column 1
rp>☘</rp>↩<rp>: 
Content model for element ruby:
See prose.

Error: Element ruby is missing a required instance of one or more of the following child elements: rt, rtc.
From line 10, column 2; to line 10, column 5
p>☘</rp>↩<rp>: </rp
Content model for element ruby:
See prose.

Error: Element ruby is missing a required instance of one or more of the following child elements: rp, rt, rtc.
From line 12, column 1; to line 12, column 7
rp>.</rp>↩</ruby>↩↩</bo
Content model for element ruby:
See prose.
```

**Comment by sideshowbarker** [comment]
What’s the expected result, per spec, for the document? No errors?

**Comment by mtrootyy** [comment]
The expected result is no errors.

**Comment by sideshowbarker** [comment]
Thanks for reporting this — I’ve pushed the fix to https://validator.w3.org/nu/ and new dev builds of the packaged checker releases will be available in a few minutes at https://github.com/validator/validator/releases
